public/app/plugins/datasource/elasticsearch/utils.ts

```diff
@@ -118,3 +118,28 @@ export const coerceESVersion = (version: string | number): string => {
       return '5.0.0';
   }
 };
+
+export const isDeprecatedVersion = (version: string): boolean => {
+  const versionArray = version.split('.').map((v) => Number(v));
```

Comment:
at other places in the elasticsearch code the `semver` package is used, i think you could do something like:

```ts
import {major, minor} from 'semver'`

if (major(version) >= 8) {
   //something
}
```

or even
```ts
import {lt} from 'semver'

if (lt(version, "7.10.0")) {
  //something
}
```

what do you think?